tests/validators/rules/test_rules_validator.py

```diff
@@ -207,14 +207,84 @@ def test_validate_count_operator_non_checkbox_answer():
     validator.validate()
 
     expected_error = {
-        "message": validator.COUNT_OPERATOR_REFERENCES_NON_CHECKBOX_ANSWER,
-        "origin_id": ORIGIN_ID,
-        "value_source": {"source": "answers", "identifier": "array-answer"},
+        "argument_type": "string",
+        "argument_value": {"identifier": "array-answer", "source": "answers"},
+        "message": validator.INVALID_ARGUMENT_TYPE_FOR_OPERATOR,
+        "operator": "count",
+        "origin_id": "block-id",
+        "valid_types": ["array"],
     }
 
     assert validator.errors == [expected_error]
 
 
+def test_validate_sum_operator():
+    sum_operator = {"+": [{"source": "answers", "identifier": "array-answer"}]}
```

Comment:
Although this is a test, we should use realistic valid format, so I would add another answer source or number to this, so the array is 2 in length.